.github/workflows/publish.yml

```diff
@@ -4,8 +4,9 @@ on:
   push:
     branches: # Snapshots
       - main
-      - 2.8.x
```

Comment:
Adding 2.8.x here does not make any sense, it only makes sense in the 2.8.x branch itself.